CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # CHANGELOG
 
+## 2.5.0 (2023-07-13)
+
+MLflow 2.5.0 includes several major features and improvements
+
+Features:
+
+- [] MLflow Gateway (#8694, @harupy)
+- [Docs / Examples / Models] Support chains with unserializable components (#8736, @liangz1)
+- [Models] [Support inference params-3]Add inference params for all flavors (#8974, @serena-ruan)
+- [Scoring] Infer spark udf return type from model output schema (#8934, @WeichenXu123)
+- [Models] [Support inference params-2]Add `params` for PyFunc predict (#8973, @serena-ruan)
+- [Models] [Support inference params-1]Add ParamSchema and ParamSpec in model signature (#8972, @serena-ruan)
+- [] Add deprecation warning in gluon doc (#8971, @harupy)
+- [Models] Coerce input examples into PyFunc compatible format to use for signature inference (#8860, @jerrylian-db)
+- [Models] Implement automatic signature inference for other flavors (#8782, @jerrylian-db)
+- [Models] Improve transformers signature inference implementation (#8795, @jerrylian-db)
+- [Models] Infer signature from input example when logging or saving models (#8725, @jerrylian-db)
```

Comment:
All of these can be collapsed into a single entry about adding support for automated signature inference :)

Reply:
makes sense

Reply:
https://github.com/mlflow/mlflow/pull/9061/files#r1261912173